src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -121,6 +121,7 @@
 
     private View mLeftFragmentContainer;
     private View mRightFragmentContainer;
+    private View shadowView;
```

Comment:
Please use **m** as prefix since is a non-public and non-static field, that is **mShadowView**